conf/sample.ini

```diff
@@ -293,6 +293,8 @@ log_queries =
 ;header_property = username
 ;auto_sign_up = true
 ;ldap_sync_ttl = 60
+;json_header = false
```

Comment:
think this option would be cool this worked the same as the headers option
json_header = Login:preferred_username Email:mail Name:full_name

So it could map more properties to the Grafana user in one setting.

Reply:
Hi @torkelo, Antti is not able to continue this pull request, so I will do a new one, but before doing that I want to confirm that following changes are what you want here: 

;json_header = false
;json_property_login = preferred_username
;json_property_email = mail
;json_property_name = full_name

Is that correct?